libs/components/pages/src/lib/modules/link-as/link-as.pipe.ts

```diff
@@ -34,4 +40,10 @@ export class LinkAsPipe implements PipeTransform {
         return false;
     }
   }
+
+  private isSkyActionHubNeedsAttention(
+    item: any
```

Comment:
This should be more strongly-typed. If it represents an object that can have a `title` property, then use this:

```
item: { title?: string } | undefined
```